resources/assets/js/services/utils.js

```diff
@@ -26,4 +26,36 @@ export default {
 
         return (h === '00' ? '' : h + ':') + i + ':' + s;
     },
+
+    /**
```

Comment:
You can access the path parser as `Vue.parsers.path`

So you can do `Vue.parsers.path.getPath(...)`